opencog/truthvalue/AttentionValue.h

```diff
@@ -60,10 +60,10 @@ class AttentionValue
     static const lti_t DEFAULTATOMLTI;   //!< long-term importance default
     static const vlti_t DEFAULTATOMVLTI; //!< very long-term default
 
-    static const sti_t MAXSTI = SHRT_MAX;
-    static const lti_t MAXLTI = SHRT_MAX;
-    static const sti_t MINSTI = SHRT_MIN;
-    static const lti_t MINLTI = SHRT_MIN;
+    static constexpr sti_t MAXSTI = SHRT_MAX;
+    static constexpr lti_t MAXLTI = SHRT_MAX;
+    static constexpr sti_t MINSTI = SHRT_MIN;
+    static constexpr lti_t MINLTI = SHRT_MIN;
```

Comment:
@linas I think SHRT_MAX and MAX are fine for now as the total STI funds being used are less SHRT_MAX.